optimizer-hints.md

```diff
@@ -813,6 +781,39 @@ In the following example, the Plan Cache is forcibly disabled when executing the
 prepare stmt from 'select  /*+ IGNORE_PLAN_CACHE() */ * from t where t.id = ?';
 ```
 
+### SET_VAR(VAR_NAME=VAR_VALUE)
+
+You can temporarily modify the value of system variables during statement execution by using the `SET_VAR(VAR_NAME=VAR_VALUE)` hint. After the statement is executed, the value of the system variable in the current session is automatically restored to the original value. This hint can be used to modify some system variables related to the optimizer and executor. For a list of system variables that can be modified using this hint, refer to [System variables](/system-variables.md).
+
+> **Warning:**
+>
+> - It is strongly recommended not to modify variables that are not explicitly supported, as this might cause unpredictable behavior.
+> - Do not write `SET_VAR` in subqueries. Otherwise it might not work. For more information, see [`SET_VAR` does not work in subqueries](#set_var-does-not-work-in-subqueries).
```

Comment:
```suggestion
> - Do not write `SET_VAR` in subqueries. Otherwise it might not take effect. For more information, see [`SET_VAR` does not take effect when written in subqueries](#set_var-does-not-take-effect-when-written-in-subqueries).
```